src/main/resources/db/migration/V20220413_164.2__HMAN-164-CreatePartyRelationshipDetails.sql

```diff
@@ -0,0 +1,25 @@
+CREATE SEQUENCE public.party_relationship_details_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+CREATE TABLE public.party_relationship_details (party_relationship_details_id bigint not null default nextval('public.party_relationship_details_id_seq'::regclass),
+                                             source_tech_party_id bigint not null,
+                                             target_tech_party_id bigint not null,
+                                             relationship_type varchar(2000)
```

Comment:
```suggestion
                                             relationship_type varchar(10)
```
This was supposed to be the new length in HMAN-90

Reply:
Fixed in https://github.com/hmcts/hmc-cft-hearing-service/pull/153/commits/165c047e535d05e348e3cb68bb5289bac865701a